src/style.rs

```diff
@@ -327,6 +335,8 @@ impl ProgressStyle {
                         continue;
                     }
 
+                    buf = buf.replace('\t', &" ".repeat(self.tab_width));
```

Comment:
It would be nice to avoid extra allocations here in the common case where there are no tabs at all.